frontend/ManagementController.cs

```diff
@@ -91,19 +91,12 @@ private async Task CreateNewMetricsOutput(IAzure azure, string rg, string aci_na
             var la_resource_Id = $"/subscriptions/{azure.SubscriptionId}/resourcegroups/{rg}/providers/microsoft.operationalinsights/workspaces/{la_name}";
             var resourceId = $"/subscriptions/{azure.SubscriptionId}/resourcegroups/{rg}/providers/Microsoft.ContainerInstance/containerGroups/{aci_name}";
 
-            try
-            {
-                await azure.DiagnosticSettings
-                                .Define($"{aci_name}metricsoutput")
-                                .WithResource(resourceId)
-                                .WithLogAnalytics(la_resource_Id)
-                                    .WithMetric("AllMetrics", TimeSpan.FromMinutes(1), 7)
-                                .CreateAsync();
-            }
-            catch(Exception ex)
-            {
-                _log.LogError(ex, "Azure error");
-            }
+            await azure.DiagnosticSettings

```

Comment:
Why remove the `try/catch`

Reply:
it was there for debugging... TODO item exists for handling errors gracefully, in a more holistic manner